docs/components/Chip/Web.stories.tsx

```diff
@@ -0,0 +1,186 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Chip } from "@jobber/components/Chip";
+import { Content } from "@jobber/components/Content";
+import { Icon } from "@jobber/components/Icon";
+
+export default {
+  title: "Components/Selections/Chip/Web",
+  component: Chip,
+  subcomponents: { Chip },
+  parameters: {
+    viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@jobber/components/Chip": ["Chip"],
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof Chip>;
+
+const BasicTemplate: ComponentStory<typeof Chip> = props => {
+  return (
+    <Content>
+      <Chip {...props} onClick={() => alert("you clicked me!")} />
+    </Content>
+  );
+};
+
+const SuffixTemplate: ComponentStory<typeof Chip> = props => {
+  return (
+    <Content>
+      <Chip {...props} onClick={() => alert("you clicked me!")}>
+        <Chip.Suffix>
+          <Icon name="cross" size="small" />
+        </Chip.Suffix>
+      </Chip>
+    </Content>
+  );
+};
+
+const PrefixTemplate: ComponentStory<typeof Chip> = props => {
+  return (
+    <Content>
+      <Chip {...props} onClick={() => alert("you clicked me!")}>
+        <Chip.Prefix>
+          <Icon name="cross" size="small" />
+        </Chip.Prefix>
+      </Chip>
+    </Content>
+  );
+};
+
+export const Base = BasicTemplate.bind({});
+Base.args = {
+  label: "Chip Label",
+  heading: "",
+  ariaLabel: "Accessible Label",
+  role: "button",
+  tabIndex: 0,
+  variation: "base",
+  disabled: false,
+  invalid: false,
+};
```

Comment:
We could do, and I will in an update early next week (I need to move Chips out) I haven't done it yet, just because I'm tight on time today. Great feedback, I've tracked it. :)

Reply:
I lied about next week. I went ahead and did this. ;) It was a good change request. Thanks Rodrigo!